README.md

```diff
@@ -71,6 +77,26 @@ Talks directly to Container Runtime Interface ([CRI](https://kubernetes.io/docs/
    kubectl logs -n prefetch-images daemonset/my-images -c prefetch
    ```
 
+6. If metrics collection was requested, wait for the endpoint to appear, and fetch them:
```

Comment:
nit: since it's copied in e2e how about extracting this into a script so it will be a live code https://github.com/stackrox/image-prefetcher/pull/18/files#diff-d0777657fa3fd81d23aaf7273e58aee453b04e67882517900c56daeef9b3e4c1R165

Reply:
I'll take a stab at this after https://github.com/stackrox/image-prefetcher/issues/17 which might reduce this to (nearly) a one-liner.